app/components/failed_network_action/index.js

```diff
@@ -25,13 +24,27 @@ export default class FailedNetworkAction extends PureComponent {
             defaultMessage: 'No internet connection',
         },
         errorDescription: {
-            id: t('mobile.failed_network_action.description'),
-            defaultMessage: 'There seems to be a problem with your internet connection. Make sure you have an active connection and try again.',
+            id: t('mobile.failed_network_action.shortDescription'),
+            defaultMessage: '{type} will load when you have an internet connection or {refresh}.',
```

Comment:
a bit concern on translations here, at lease in spanish I would use an article before the word "messages" but is not really clear that I would need one here. I get that you need the `Try again` action, maybe think about passing the action as a prop instead of the whole `<FormattedText>` element or some other way